packages/venia-ui/lib/drivers/adapter.js

```diff
@@ -116,8 +117,27 @@ const VeniaAdapter = props => {
  * other modules in the codebase need access to it.
  */
 VeniaAdapter.apolloLink = apiBase => {
+    // Intercept and shrink URLs from GET queries. Using
+    // GET makes it possible to use edge caching in Magento
+    // Cloud, but risks exceeding URL limits with default usage
+    // of Apollo's http link. `shrinkGETQuery` encodes the URL
+    // in a more efficient way.
+    const customFetchToShrinkQuery = (uri, options) => {
+        let url = uri;
+        if (options.method === 'GET') {
+            url = shrinkGETQuery(uri);
```

Comment:
Is there any reason we wouldn't want to strip the ignored characters for POST as well?

Reply:
We could! I honestly hadn't given it much thought 😄  Was just critical for GET so we don't exceed limits